jedi/evaluate/docstrings.py

```diff
@@ -29,11 +29,13 @@
     r'\s*:type\s+%s:\s*([^\n]+)',  # Sphinx
     r'\s*:param\s+(\w+)\s+%s:[^\n]+',  # Sphinx param with type
     r'\s*@type\s+%s:\s*([^\n]+)',  # Epydoc
+    '\s*%s\s*\((.+?)\):.+',  # Google param with type
```

Comment:
>  it's basically a context free language

I understand that, but you could check it before hand. That makes it a little bit more complicated (in the Python area), but could still work. Obviously modifying regexes would not be enough anymore.